docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1134,6 +1138,41 @@ Result:
 └──────────────────────────────────────────────────────────────┘
 ```
 
+## addDate
+
+Adds the time interval or date interval to the provided date or date with time.
```

Comment:
I think we should also refer the reader to the existing function `dateAdd` and vice versa.